bokeh/models/tools.py

```diff
@@ -392,6 +392,41 @@ class BoxZoomTool(Drag):
 
     """)
 
+class ZoomInTool(Action):
+    """ *toolbar icon*: |zoom_in_icon|
+
+    The zoom-in tool allows users to click a button to zoom in
+    by a fixed amount.
+
+    .. |zoom_in_icon| image:: /_images/icons/ZoomIn.png
+        :height: 18pt
+
+    """
+    # TODO ZoomInTool dimensions should probably be constrained to be the same as ZoomOutTool
```

Comment:
@mattpap, at the SciPy sprint when I started looking at this, I remember @bryevdv mentioning wanting to look into a way to bundle tools that should come together.  Since it impacts more than these new zoom buttons, maybe that should be broken out into its own issue.  I'd be happy to discuss and/or remove the TODO from the code.

It is also entirely possible that I'm making up this idea or that I misunderstood @bryevdv